src/reactNodeToString.js

```diff
@@ -1,4 +1,9 @@
-import reactElementToJSXString from 'react-element-to-jsx-string'
+import { shallow } from 'enzyme'
+
+function reactElementToJSXString (node) {
+  const Wrapper = () => node
+  return shallow(<Wrapper />).debug()
```

Comment:
Hey @rylanc I am not sure, if I can comment on this, but I found this while I am using your changes. Hence, bring to your notice. We need `import React from 'react'` before creating a react element here.

Reply:
eslint-plugin-react should be used to lint for this, fwiw.

Reply:
(and the airbnb eslint config has it all set up for you ;-) )